src/assets/scripts/client/InputController.js

```diff
@@ -221,16 +222,15 @@ export default class InputController {
      * @private
      */
     _addMeasurePoint(event, shouldReplaceLastPoint = false) {
-        const currentMousePosition = CanvasStageModel.translateMousePositionToCanvasPosition(
+        const mouseCanvasPos = CanvasStageModel.calculateCanvasPositionFromPagePosition(
```

Comment:
please use complete words in names `pos` -> `position`

Reply:
I was cheating for the 130 character limit for lint 😞 

I can break it into two lines I suppose.